docs/storage.md

```diff
@@ -129,3 +129,24 @@ Note that on the read path, Prometheus only fetches raw series data for a set of
 ### Existing integrations
 
 To learn more about existing integrations with remote storage systems, see the [Integrations documentation](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+
+## Backfilling from OpenMetrics format
+
+### Overview
+
+If a user wants to migrate from an existing monitoring system (source) to Prometheus, they can add their older data to TSDB easily using backfilling. The source system may be another TSDB which is capable of dumping data in [OpenMetrics](https://openmetrics.io/) format. However,one should be careful and note that it is not safe to backfill data from the last 3 hours (the current head block).
+
+Prometheus keeps at most 2 hours of data in the memory. Hence, make sure that there is enough RAM for 2 hours worth of data at a time.
+
+### Usage 
+
+Backfilling can be used via the promtool command line. Promtool will write the blocks to a directory. By default this is data/, you can change it by passing the name of the desired output directory.
+
+```
+./promtool create-blocks-from openmetrics name_of_input_file name_of_output_file
```

Comment:
Why does this say output file when it writes to a directory? Is the last argument optional?

Reply:
Yes it is optional